src/components/InlineLoading/InlineLoading-story.js

```diff
@@ -16,6 +16,10 @@ import InlineLoading from '../InlineLoading';
 
 const props = () => ({
   success: boolean('Loading successful state (success)', false),
+  iconDescription: text(
```

Comment:
Same comment about unnecessary knob. Just because it's in the prop table, doesn't mean we need to make it a knob. I went on a rant about this [https://ibm-studios.slack.com/archives/GCQ5R263T/p1571254152018700?thread_ts=1571245479.016100&cid=GCQ5R263T](on slack), let me know if you need any more context.